src/athena.hpp

```diff
@@ -69,6 +69,10 @@ struct LogicalLocation { // aggregate and POD type
   }
 };
 
+/// Defines the maximum size of the static array used in the IndexShape objects
+constexpr int NDIM = 3;
```

Comment:
Would it be better to put this in `cmake` instead of `athena`? Or maybe here is the better spot? I don't know.

Reply:
I think it is fine for now. We can move it when there is a tangible advantage later.

Reply:
Sounds good. Let's move the static assert though.